### DISCUSSION TITLE
Podman has a command with a dependency `podman machine init`

### DISCUSSION THREAD

**Discussion by georgettica** [Homebrew/discussions#1747] [Everyday usage]
this requires `brew install qemu`

should there be a popup that shows this is a soft requirement on macOS?

**Comment by SMillerDev** [comment]
Is there any documentation about this upstream?

**Reply by georgettica** [reply]
not that I gathered.. just saw the cli requested this package and then things worked better

**Reply by SMillerDev** [reply]
Actually the upstream docs mention that it won't work on non-Linux machines without a VM so I think this is kinda expected.

**Reply by georgettica** [reply]
I can see that, maybe add something in the post installation text?
